Library/Homebrew/livecheck/skip_conditions.rb

```diff
@@ -0,0 +1,228 @@
+        params(
+          formula:       Formula,
+          livecheckable: T::Boolean,
+          full_name:     T::Boolean,
+          verbose:       T::Boolean,
+        ).returns(T.nilable(Hash))
+      end
+      def formula_deprecated(formula, livecheckable, full_name: false, verbose: false)
+        return unless formula.deprecated? && !livecheckable
+
+        Livecheck.status_hash(formula, "deprecated", full_name: full_name, verbose: verbose)
+      end
+
+      sig do
+        params(
+          formula:       Formula,
+          livecheckable: T::Boolean,
+          full_name:     T::Boolean,
+          verbose:       T::Boolean,
+        ).returns(T.nilable(Hash))
+      end
+      def formula_disabled(formula, livecheckable, full_name: false, verbose: false)
+        return unless formula.disabled? && !livecheckable
+
+        Livecheck.status_hash(formula, "disabled", full_name: full_name, verbose: verbose)
+      end
+
+      sig do
+        params(
+          formula:       Formula,
+          livecheckable: T::Boolean,
+          full_name:     T::Boolean,
+          verbose:       T::Boolean,
+        ).returns(T.nilable(Hash))
+      end
+      def formula_versioned(formula, livecheckable, full_name: false, verbose: false)
+        return unless formula.versioned_formula? && !livecheckable
+
+        Livecheck.status_hash(formula, "versioned", full_name: full_name, verbose: verbose)
+      end
+
+      sig do
+        params(
+          cask:          Cask::Cask,
+          livecheckable: T::Boolean,
+          full_name:     T::Boolean,
+          verbose:       T::Boolean,
+        ).returns(T.nilable(Hash))
+      end
+      def cask_discontinued(cask, livecheckable, full_name: false, verbose: false)
+        return unless cask.discontinued? && !livecheckable
+
+        Livecheck.status_hash(cask, "discontinued", full_name: full_name, verbose: verbose)
+      end
+
+      sig do
+        params(
+          cask:          Cask::Cask,
+          livecheckable: T::Boolean,
+          full_name:     T::Boolean,
+          verbose:       T::Boolean,
+        ).returns(T.nilable(Hash))
+      end
+      def cask_version_latest(cask, livecheckable, full_name: false, verbose: false)
+        return unless cask.present? && cask.version&.latest? && !livecheckable
+
+        Livecheck.status_hash(cask, "latest", full_name: full_name, verbose: verbose)
+      end
+
+      sig do
+        params(
+          cask:          Cask::Cask,
+          livecheckable: T::Boolean,
+          full_name:     T::Boolean,
+          verbose:       T::Boolean,
+        ).returns(T.nilable(Hash))
+      end
+      def cask_url_unversioned(cask, livecheckable, full_name: false, verbose: false)
+        return unless cask.present? && cask.url&.unversioned? && !livecheckable
+
+        Livecheck.status_hash(cask, "unversioned", full_name: full_name, verbose: verbose)
+      end
+
+      # Skip conditions for formulae.
+      FORMULA_CHECKS = [
```

Comment:
```suggestion
      FORMULA_CHECKS =%i[
```

will allow you to skip `:` and `,`

Reply:
This is good to know but unfortunately we currently encounter a style error when we use the percent format here:

```
livecheck/skip_conditions.rb:159:24: C: [Correctable] Use [] for an array of symbols.
      FORMULA_CHECKS = %i[ ...
                       ^^^
livecheck/skip_conditions.rb:168:21: C: [Correctable] Use [] for an array of symbols.
      CASK_CHECKS = %i[ ...
                    ^^^
```

Reply:
Ah, ignore me then 🤦🏻